warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -175,10 +185,22 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                 browser_details = selenium_Utils.\
                     get_browser_details(browser, self.datafile, **arguments)
             if browser_details is not None:
+                try:
+                    from pyvirtualdisplay import Display
+                    display = Display(visible=0, size=(1024, 768))
+                    display.start()
+                    pNote("Running in headless mode")
```

Comment:
@navvrnd  : "pNote" default "print_type" value is "print_info". So I have used that.